cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -442,6 +459,113 @@ Replaced by:
 ----
 WHERE NOT isEmpty([1, 2, 3])
 ----
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+distance(n.prop, point({x:0, y:0})
+----
+a|
+Replaced by:
+[source, cypher, role="noheader"]
+----
+point.distance(n.prop, point({x:0, y:0})
+----
+
+a|
+label:functionality[]
+label:removed[]
+[source, cypher, role="noheader"]
+----
+point({x:0, y:0}) <= point({x:1, y:1}) <= point({x:2, y:2})
+----
+a|
+The ability to use operators `<`, `<=`, `>`, and `>=` on spatial points is removed.
```

Comment:
The ability to use operators `+<+`, `+<=+`, `+>+`, and `+>=+` on spatial points is removed.

Escape operators, especially `<=` or it will not show these characters in asciidoc.